models/Asset.php

```diff
@@ -1400,7 +1400,7 @@ public function getProperties()
     }
 
     /**
-     * @param array $properties
+     * @param Property[] $properties
```

Comment:
do we need a fix for Documents and DataObjects as well ?

Reply:
Done